.gitignore

```diff
@@ -1,5 +1,5 @@
 .idea
 
 # Added by cargo
-/target
-/Cargo.lock
+target
+Cargo.lock
```

Comment:
What about `honey-badger/target` and `honey-badger/Cargo.lock` or maybe there's a generic way like `**/target` or `**/Cargo.lock` or somehting.